src/compute-client/src/explain/text.rs

```diff
@@ -330,8 +330,15 @@ impl DisplayText<PlanRenderingContext<'_, Plan>> for Plan {
                 };
                 ctx.indented(|ctx| input.fmt_text(f, ctx))?;
             }
-            Union { inputs } => {
-                writeln!(f, "{}Union", ctx.indent)?;
+            Union {
+                inputs,
+                consolidate_output,
+            } => {
+                writeln!(
+                    f,
+                    "{}Union consolidate_output={}",
```

Comment:
Let's not print it it has the default value (`false`).

Reply:
Ok, changed it.